.github/CODEOWNERS

```diff
@@ -1,9 +1,13 @@
-# This CODEOWNERS file sets the individuals responsible for code in the sdk-docs repository.
+# This CODEOWNERS file sets the individuals responsible for code in the sdk-docs repository
 
-# These users are the default owners for everything in the repo.
-# They will be requested for review when someone opens a pull request.
+# These users are the default owners for everything in the repo
+# They will be requested for review when someone opens a pull request
 
-# You can also specify code owners for specific directories or files.
-# For example:
+* @zksync-sdk/devxp
+/content
```

Comment:
Do these specific directories need to have the GitHub Team specified? Or is it okay to leave them blank like that?

Reply:
The ruleset here is that devxp are owners for "everything" except for `/content` and `/public`. This way we ensure we review changes to the project structure but aren't required on the content and public folders so we don't hold up reviews in there. 

If txFusion should be owners for all the content in the /content directory I can also specify that.

Reply:
Agreed about the split, and also think it's a good idea for both `devxp` and `txfusion` to be listed as reviewers for the `/content` directory.

Can we also add comments to the `CODEOWNERS` file with your explanation above?